README.md

```diff
@@ -75,8 +75,10 @@ $ distro -j
 
 $ python
 >>> import distro
->>> distro.linux_distribution(full_distribution_name=False)
-('centos', '7.1.1503', 'Core')
+>>> distro.name(pretty=True)
+'CentOS Linux 8'
+>>> distro.version(best=True)
+'8.4.2105'
```

Comment:
Good idea. Let's maybe add an additional…

```
>>> distro.id()
'centos'
```

in front, to be complete.

Reply:
Cool, better now?